datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamResources.java

```diff
@@ -666,21 +666,22 @@ public UpdateResponse delete(String datastreamName) {
 
   @Override
   public Datastream get(String name) {
+    Datastream stream = null;
     try {
-      LOG.info("Get datastream called for datastream {}", name);
```

Comment:
nit: maybe we should keep this log? That Get was called? I know that I've used this log for other operations we perform (stop, resume, pause, etc), and it may be helpful to know if the server is receiving Get requests or not if we see some failures.

Reply:
Sorry, wasn't intended. Fixed it.